services/datalake/pod-datalake/src/handlers/video.ts

```diff
@@ -28,10 +30,12 @@ export async function requestHLS (ctx: MeasureContext, workspace: string, name:
     return
   }
   const streamReq: StreamRequest = { format: 'hls', source: name, workspace }
+  const token = generateToken(systemAccountUuid)
```

Comment:
recommend to add `extra` with service name so we know which service makes requests

Reply:
what is extra? I think we need to set aud/issuer

Reply:
See `generateToken` function:
```
export function generateToken (
  accountUuid: PersonUuid,
  workspaceUuid?: WorkspaceUuid,
  extra?: Record<string, string>,
  secret?: string
): string {
  ...
}
```

Reply:
ok, I see extra is a part of payload. Will fix :)